### PR TITLE
Added links to hero section buttons

### PR DESCRIPTION
**Pull request by SaranshBangar**
### Description

1. Added links to hero section buttons
2. Added a new router link for components page

### Checklist

- [X] I have tested these changes locally.
- [X] My code follows the project's coding guidelines.
- [X] I have updated the documentation, if necessary.
- [X] I have added tests to cover my changes.

### Additional Notes

Resolves issue #19

### CODE REVIEW

File: src/components/Home/HeroSection.tsx [hunk 1]
```diff
@@ -36,17 +36,16 @@ const HeroSection = () => {
...
-            // onClick={() =>
-            //   alert(`${window.screen.width} ${window.screen.height}`)
-            // }
-          >
-            Explore Components
-          </button>
-          <button className="z-10 min-w-[12rem] rounded-md border px-4 py-2">
-            Get Started
-          </button>
+          <a href="/components" target='_blank'>
```
Comment (chandel-aman) [inline src/components/Home/HeroSection.tsx]: Why use `anchor` tag?
Use `Link` component instead
Also we don't want to open the pages in new tab

**Review by chandel-aman — CHANGES_REQUESTED** [review]

File: src/components/Home/HeroSection.tsx (hunk 1 above)
Comment (SaranshBangar) [reply src/components/Home/HeroSection.tsx]: Ok sir, I will make the necessary changes

Comment (SaranshBangar) [reply src/components/Home/HeroSection.tsx]: I have made the necessary changes sir
1. Changed `<a>` tags to `Link` tags
2. Made sure that the pages do not open in a new tab

**Comment by AkashSharma001** [comment]
@SaranshBangar 
Pull latest changes from the 'develop' branch and add links to every 'Get Started' button.

**Comment by SaranshBangar** [comment]
> @SaranshBangar Pull latest changes from the 'develop' branch and add links to every 'Get Started' button.

Understood sir, for that I would need to delete my current forked repo and raise a new PR, please give me a few mins until I sort it out

**Pull request closed without merging**